### ISSUE TITLE
WHY only 1 CPU when hmmsearch 

### ISSUE THREAD

**Issue by PonderLi** [AnantharamanLab/METABOLIC#182]
Hi, thanks for so helpful tools!
I noticed that when using hmmsearch, the "--cpu" is always 1, could I increase it to speed up the analysis?

**Comment by snpone** [comment]
no, you can't. Cause ` %result_hmm` only dealing ko's and store the results one by one. In fact, the other step hmmsearch can run multi-threads. 
So don't worry, just be patient.
